metrics/web/test/common/presentation/navigation/route_configuration/route_configuration_test.dart

```diff
@@ -1,76 +1,254 @@
 // Use of this source code is governed by the Apache License, Version 2.0
 // that can be found in the LICENSE file.
 
+import 'package:flutter/widgets.dart';
 import 'package:metrics/common/presentation/navigation/route_configuration/route_configuration.dart';
 import 'package:metrics/common/presentation/navigation/route_configuration/route_name.dart';
 import 'package:test/test.dart';
 
-import '../../../../test_utils/matchers.dart';
-
 void main() {
   group("RouteConfiguration", () {
-    const name = RouteName.login;
-    const path = 'test_path';
-    const authorizationRequired = false;
     const parameters = {'test': 'test'};
 
     test(
-      "throws an AssertionError if the given name is null",
+      ".loading() creates an instance with the the correct name",
       () {
+        const expectedName = RouteName.loading;
+
+        const configuration = RouteConfiguration.loading();
+
+        expect(configuration.name, equals(expectedName));
+      },
+    );
+
+    test(
+      ".loading() creates an instance with the correct path",
+      () {
+        const expectedPath = Navigator.defaultRouteName;
+
+        const configuration = RouteConfiguration.loading();
+
+        expect(configuration.path, equals(expectedPath));
+      },
+    );
+
+    test(
+      ".loading() creates an instance with the correct authorization required value",
+      () {
+        const expectedAuthorizationRequired = false;
+
+        const configuration = RouteConfiguration.loading();
+
         expect(
-          () => RouteConfiguration(name: null, authorizationRequired: true),
-          throwsAssertionError,
+          configuration.authorizationRequired,
+          equals(expectedAuthorizationRequired),
         );
       },
     );
 
     test(
-      "throws an AssertionError if the given authorization required is null",
+      ".loading() creates an instance with the parameters map equal to null",
+      () {
+        const configuration = RouteConfiguration.loading();
+
+        expect(configuration.parameters, isNull);
+      },
+    );
+
+    test(
+      ".login() creates an instance with the the correct name",
```

Comment:
🏷️ 
Please apply in other testcases where necessary.
```suggestion
      ".login() creates an instance with the correct name",
```